packages/web/src/components/gcds-file-uploader/gcds-file-uploader.tsx

```diff
@@ -333,6 +333,47 @@ export class GcdsFileUploader {
     this.internals.setFormValue(formData);
   };
 
+  /*
+   * Handle drop event on file uploader
+   */
+  private handleDrop(e) {
+    e.preventDefault();
+
+    const droppedFiles = e.dataTransfer.files;
+
+    if (droppedFiles.length > 0) {
```

Comment:
Small suggestion: `droppedFiles` could be `null`, let's ensure we check for it in the if statement.

```suggestion
    if (droppedFiles && droppedFiles.length > 0) {
```